packages/manager/cypress/e2e/core/account/user-verification-banner.spec.ts

```diff
@@ -0,0 +1,236 @@
+import { profileFactory, securityQuestionsFactory } from '@src/factories';
+import { accountUserFactory } from '@src/factories/accountUsers';
+import { grantsFactory } from '@src/factories/grants';
+import {
+  mockGetUser,
+  mockGetUserGrants,
+  mockGetUsers,
+} from 'support/intercepts/account';
+import { mockGetSecurityQuestions } from 'support/intercepts/profile';
+import {
+  mockAppendFeatureFlags,
+  mockGetFeatureFlagClientstream,
+} from 'support/intercepts/feature-flags';
+import { ui } from 'support/ui';
+import { mockGetProfile } from 'support/intercepts/profile';
+import { makeFeatureFlagData } from 'support/util/feature-flags';
+import { verificationBannerNotice } from 'support/constants/user';
+
+describe('User verification banner', () => {
+  /*
+   * - Confirms that a banner is present the child users do not have a phone number or security questions.
+   * - Confirms that the "Add Verification Details" button redirects the user to /profile/auth.
+   */
+  it('can show up when a child user has not associated a phone number or set up security questions for their account', () => {
+    const mockChildProfile = profileFactory.build({
+      username: 'child-user',
+      user_type: 'child',
+      verified_phone_number: null,
+    });
+
+    const mockChildUser = accountUserFactory.build({
+      restricted: false,
+      username: 'child-user',
+      user_type: 'child',
+      verified_phone_number: null,
+    });
+
+    const mockRestrictedProxyUser = accountUserFactory.build({
+      restricted: true,
+      user_type: 'proxy',
+      username: 'restricted-proxy-user',
+      verified_phone_number: null,
+    });
+
+    const mockUserGrants = grantsFactory.build({
+      global: { account_access: 'read_write' },
+    });
+
+    // TODO: Parent/Child - M3-7559 clean up when feature is live in prod and feature flag is removed.
+    mockAppendFeatureFlags({
+      parentChildAccountAccess: makeFeatureFlagData(true),
+    }).as('getFeatureFlags');
+    mockGetFeatureFlagClientstream().as('getClientStream');
+
+    mockGetUsers([mockRestrictedProxyUser]).as('getUsers');
```

Comment:
Nit: if we're not using the aliases in a `cy.wait` call, we could remove them here. (Related: I don't know if not waiting for feature flags could potentially cause any flakiness, which is why we generally do it, or if we are confident it will happen fast enough that it's not very necessary in the first place.)

```suggestion
    mockAppendFeatureFlags({
      parentChildAccountAccess: makeFeatureFlagData(true),
    });
    mockGetFeatureFlagClientstream();

    mockGetUsers([mockRestrictedProxyUser]);
```